internal/provider/schema.go

```diff
@@ -431,11 +439,13 @@ func (ksm *k0sctlSchemaModel) Cluster(ctx context.Context) (k0sctl_v1beta1.Clust
 					SSH: &k0s_rig.SSH{
 						Address: shssh.Address.ValueString(),
 						KeyPath: shssh.KeyPath.ValueStringPointer(),
+						HostKey: shssh.KeyContent.ValueString(),
 						User:    shssh.User.ValueString(),
 						Port:    int(shssh.Port.ValueInt64()),
 						Bastion: &k0s_rig.SSH{
 							Address: shssh.Bastion[0].Address.ValueString(),
 							KeyPath: shssh.Bastion[0].KeyPath.ValueStringPointer(),
+							HostKey: shssh.KeyContent.ValueString(),
```

Comment:
```suggestion
							HostKey: shssh.Bastion[0].KeyContent.ValueString(),
```
need to use the bastion values here, not the ssh values